lib/charms/opensearch/v0/opensearch_plugin_manager.py

```diff
@@ -50,399 +39,185 @@
 
 logger = logging.getLogger(__name__)
 
+if TYPE_CHECKING:
+    from charms.opensearch.v0.opensearch_base_charm import OpenSearchBaseCharm
 
-ConfigExposedPlugins = {
-    "opensearch-knn": {
-        "class": OpenSearchKnn,
-        "config": "plugin_opensearch_knn",
-    },
-}
+SMTP_SECRET_LABEL = "notifications-plugin"
```

Comment:
could we invert this for better traversal:
```suggestion
SMTP_SECRET_LABEL = "plugin-notifications"
```